scripts/qa_crowbarsetup.sh

```diff
@@ -415,7 +415,7 @@ function add_ha_repo()
         fi
         # Note no zypper alias parameter here since we don't want to
         # zypper addrepo on the admin node.
-        add_mount "$repo" "clouddata.cloud.suse.de:/srv/nfs/repos/$repo" \
+        add_mount "$repo/sle-11-x86_64" "clouddata.cloud.suse.de:/srv/nfs/repos/$repo" \
```

Comment:
shouldn't it be ```sle-12-x86_64```? SES is only available for SLES12. I think it's a bug in ```sync-repos``` which uses ```sle-11-x86_64``` as well.